src/components/VisualizationOptions/Options/ColTotals.js

```diff
@@ -0,0 +1,14 @@
+import i18n from '@dhis2/d2-i18n'
+import React from 'react'
+import CheckboxBaseOption from './CheckboxBaseOption'
```

Comment:
I see many of these lint warnings lately.
Should we import with brackets instead?
```suggestion
import { CheckboxBaseOption } from './CheckboxBaseOption'
```